ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -266,6 +266,9 @@ def flatten(self, order="C"):
         else:
             return np_frontend.ravel(self._ivy_array, order="C")
 
+    def getfield(self, dtype, offset=0):
+        return np_frontend.getfield(self._ivy_array, dtype, offset).astype(dtype)
```

Comment:
Hey this function isn't implemented on the numpy frontend
You'll have to implement it using the composition of ivy functions